README.md

```diff
@@ -77,6 +77,7 @@ release.
 | Tracing SDK                 | Alpha   | September 30 2019 |
 | Metrics API                 | Alpha   | September 30 2019 |
 | Metrics SDK                 | Alpha   | September 30 2019 |
+| B3 Trace Exporter           | Alpha   | September 30 2019 |
```

Comment:
I guess `B3` is the propagation format, here we want to use `Zipkin Trace Exporter`?

Reply:
Ref: https://zipkin.io/pages/tracers_instrumentation